washtub/http/http.go

```diff
@@ -0,0 +1,113 @@
+package washtubhttp
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/nsqsink/sink/entity"
+	"github.com/nsqsink/sink/washtub"
+)
+
+type Client struct {
+	// The `stopPulse` channel of type `chan struct{}` is used to signal the client to stop pulsing. It is
+	// a synchronization mechanism that allows the client to gracefully stop its pulsing operation. When a
+	// value is sent on this channel, it indicates that the client should stop pulsing and exit.
+	stopPulse chan struct{}
+
+	// The `pulseInterval time.Duration` is a field in the `Client` struct that represents the interval at
+	// which the client should send pulses. It is of type `time.Duration`, which allows you to specify the
+	// duration of the interval in a human-readable format, such as seconds, minutes, or hours. This field
+	// determines how often the client should send pulses to a server or perform some other periodic task.
+	pulseInterval time.Duration
+
+	// The `address string` field in the `Client` struct is used to store the address of the server that
+	// the client will send requests to. It represents the URL or IP address of the server. This field is
+	// used in the `call` method to create an HTTP request to the specified server address.
+	address string
+
+	// client is the HTTP client to use.
+	client *http.Client
+}
+
+func NewWashtuber(ctx context.Context, address string) (washtub.Washtuber, error) {
```

Comment:
gw jadi ada ide, ini kan client washtub, dibikin pisah repo aja? wkwk

Reply:
kyknya timeout sama pulse intervalnya bisa diconfig in deh coach

Reply:
consumer kan bisa banyak, tp init cuma sekali. mau gimana simpan channel id nya?